src/amo/components/Link.js

```diff
@@ -0,0 +1,27 @@
+import React, { PropTypes } from 'react';
+import { compose } from 'redux';
+import { connect } from 'react-redux';
+import { Link } from 'react-router';
+
+
+export class InternalLinkBase extends React.Component {
```

Comment:
oh, whoops, nevermind. I see that it needs to work with `Link` from `react-router`